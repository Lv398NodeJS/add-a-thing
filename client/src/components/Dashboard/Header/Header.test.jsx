import React from 'react';
// import { shallow } from 'enzyme';
// import Header from './Header';

// describe('Header', () => {
//   it('when the user is logged in Header should have only Sign Out button', () => {
//     const isLoggedIn = true
//     const component = shallow(<Header isLoggedIn={isLoggedIn} />);
//     expect(component).toMatchSnapshot();
//   });
//
//   it('when the user is not logged in Header should have Log in and Sign Up buttons', () => {
//     const isLoggedIn = false
//     const component = shallow(<Header isLoggedIn={isLoggedIn} />);
//     expect(component).toMatchSnapshot();
//   });
// });
