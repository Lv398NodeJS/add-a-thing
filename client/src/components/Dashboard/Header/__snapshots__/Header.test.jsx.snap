// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar when the user is logged in Header should have only Sign Out button 1`] = `ShallowWrapper {}`;

exports[`NavBar when the user is not logged in Header should have Log in and Sign Up buttons 1`] = `ShallowWrapper {}`;
